README.md

```diff
@@ -85,7 +85,7 @@ sizes near the tails. We now have 4 scale functions supported for both
 major digest forms (`MergingDigest` and `AVLTreeDigest`) to allow
 different trade-offs in terms of accuracy.
 
-These scale functions now have associated proofs that they all
+These scale functions now have associated proof that they all
```

Comment:
I think that there are multiple proofs (one each for each scale function)

What is the rationale for the singular here? I may be off base.